src/main/java/com/endava/soa_5/actions/LogInAction.java

```diff
@@ -0,0 +1,23 @@
+package com.endava.soa_5.actions;
+
+import com.endava.soa_5.Driver;
+import com.endava.soa_5.constant.UsersCredentials;
+import com.endava.soa_5.page_objects.LogInPageObject;
+
+public class LogInAction  {
+    LogInPageObject logInPageObject = new LogInPageObject(Driver.getInstance().getDriver());
+
+    public void login(UsersCredentials user) {
+        logInPageObject.getLogInLink().click();
```

Comment:
webelement waiters will be implemented in others PRs ?

Reply:
yep. in the next PR, the waiters will be added